dhis-2/dhis-test-web-api/src/test/java/org/hisp/dhis/webapi/controller/deduplication/DeduplicationControllerTest.java

```diff
@@ -66,38 +70,37 @@ class DeduplicationControllerTest extends H2ControllerIntegrationTestBase {
   private OrganisationUnit orgUnit;
   private TrackedEntityType trackedEntityType;
   private TrackedEntity origin;
-  private TrackedEntity duplicate1;
+  private TrackedEntity duplicate;
   private PotentialDuplicate potentialDuplicate1;
   private PotentialDuplicate potentialDuplicate2;
+  private User user;
 
   @BeforeEach
   void setUp() {
+    injectAdminIntoSecurityContext();
     orgUnit = createOrganisationUnit(CodeGenerator.generateUid());
     dbmsManager.save(orgUnit);
+    getAdminUser().getOrganisationUnits().add(orgUnit);
 
-    trackedEntityType = createTrackedEntityType('A');
-    dbmsManager.save(trackedEntityType);
-
-    origin = createTrackedEntity(orgUnit, trackedEntityType);
-    duplicate1 = createTrackedEntity(orgUnit, trackedEntityType);
-    TrackedEntity duplicate2 = createTrackedEntity(orgUnit, trackedEntityType);
+    trackedEntityType = createAndSaveTrackedEntityType('A');
+    origin = createAndSaveTrackedEntity(orgUnit, trackedEntityType);
+    duplicate = createAndSaveTrackedEntity(orgUnit, trackedEntityType);
+    TrackedEntity duplicate2 = createAndSaveTrackedEntity(orgUnit, trackedEntityType);
 
-    dbmsManager.save(origin);
-    dbmsManager.save(duplicate1);
-    dbmsManager.save(duplicate2);
-
-    potentialDuplicate1 = new PotentialDuplicate(UID.of(origin), UID.of(duplicate1));
+    potentialDuplicate1 = new PotentialDuplicate(UID.of(origin), UID.of(duplicate));
     save(potentialDuplicate1);
     potentialDuplicate2 = new PotentialDuplicate(UID.of(origin), UID.of(duplicate2));
     save(potentialDuplicate2);
+
+    user = createAndAddUser("username");
```

Comment:
```suggestion
    user = createAndAddUser("marceline");
```

Using the field name as the value is confusing IMHO

Reply:
ok, how about "regular-user"?